cassandra/io/asyncioreactor.py

```diff
@@ -162,7 +169,7 @@ def push(self, data):
         else:
             chunks = [data]
 
-        if self._loop_thread.ident != get_ident():
+        if self._loop_thread != threading.current_thread():
```

Comment:
+1 Since ` self._loop_thread != threading.current_thread()` is a common and direct way to verify that the current thread is not the specified thread, no matter how the thread idents are typed, created, stored, updated.